[next] Cannot resolve module 'jss-preset-default'

### Description

```
ERROR in /Users/kross/projects/material-ui-next/src/styles/MuiThemeProvider.js
Module not found: Error: Cannot resolve module 'jss-preset-default' in /Users/kross/projects/material-ui-next/src/styles
 @ /Users/kross/projects/material-ui-next/src/styles/MuiThemeProvider.js 16:24-53
```

### Images & references
ac18a75b0d20f4f1b3991364c20d76a57c5c0d89

### Problem description
Removed necessary dependency.

### Link to minimally-working code that reproduces the issue

docs site

--- a/package.json
+++ b/package.json
@@ -50,6 +50,7 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "dom-helpers": "^3.0.0",
+    "jss-preset-default": "^0.9.0",
     "jss-theme-reactor": "^0.6.0",
     "keycode": "^2.1.1",
     "lodash": "^4.17.2",

